src/bootstrap/native.rs

```diff
@@ -196,6 +196,13 @@ impl Step for Llvm {
             cfg.define("LLVM_BUILD_32_BITS", "ON");
         }
 
+        if builder.config.lldb_enabled {
+            cfg.define("LLVM_EXTERNAL_CLANG_SOURCE_DIR", builder.src.join("src/tools/clang"));
+            cfg.define("LLVM_EXTERNAL_LLDB_SOURCE_DIR", builder.src.join("src/tools/lldb"));
+        } else {
+            cfg.define("LLVM_ENABLE_LIBXML2", "OFF");
```

Comment:
Could you add a comment why this isn't done on the previous branch?

Reply:
It took a little digging but I found https://github.com/rust-lang/rust/pull/50104; I'll link to that.

Reply:
Yeah, I only knew to make this change because the lldb build failed without it.  Here's the comment in my local tree now:

```
            // LLDB requires libxml2; but otherwise we want it to be disabled.
            // See https://github.com/rust-lang/rust/pull/50104
            cfg.define("LLVM_ENABLE_LIBXML2", "OFF");
```